README.md

```diff
@@ -52,6 +53,11 @@ Search links that point directly to suitable issues to contribute to on GitHub
 - [GitHub Learning Resources](https://help.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources
 - [Pro Git](https://git-scm.com/book/en/v2) - The entire Pro Git book, written by Scott Chacon and Ben Straub and published by Apress.
 - [Try Git](https://try.github.io/) - Learn Git in your browser for free
+- [Udacity](https://www.udacity.com/course/how-to-use-git-and-github--ud775) - Free git online course
+
+## Online tutorials
+- [Codecademy](https://www.codecademy.com/) - Learn to code
```

Comment:
Please remove this section. Knowing how to code is necessary to contribute to open source projects. But that can be solved by other sources. This is focused on getting over the obstacle of contributing to open source.